### ISSUE TITLE
Cannot detect DistZip with dependencies

### ISSUE THREAD

**Issue by illeatmyhat** [cloudfoundry/java-buildpack#597]
I'm attempting to deploy [Grobid](https://github.com/kermitt2/grobid), which uses Gradle to create a DistZip.
As per the instructions, I perform a `./gradlew clean assemble` which produces DistZips for its various subprojects.
The subproject *grobid-service* depends on *grobid-home*, so I extract their DistZips to the same folder, so the resulting directory structure looks like
```
/
  grobid-home
  grobid-service
```

Attempting to push this folder using `cf push app_name -b https://github.com/cloudfoundry/java-buildpack` results in the error
```
ERROR Compile failed with exception #<RuntimeError: No container can run this application. Please ensure that you've pushed a valid JVM artifact or artifacts using the -p command line argument or path manifest entry.
```
The same error occurs when uploading a zip using the -p argument.

tl;dr DistZip detection seems to fail when one project relies on another project. Maybe make it possible to point out where the /bin and /lib are.
I also need to be able to provide parameters to the startup script.

**Comment by nebhale** [comment]
With the advent of [Cloud Native Buildpacks](https://buildpacks.io) we're unlikely to make changes to the existing behavior of this buildpack.  Instead, newer implementations without the previous restrictions will be made there.  We're closing this issue out in lieu of a new implementation.
